components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -128,40 +140,63 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
             ChannelComponent = SidebarGroupChannel;
         }
 
-        return (
-            <Draggable
-                draggableId={channel.id}
-                index={channelIndex}
-            >
-                {(provided, snapshot) => {
-                    return (
-                        <li
-                            draggable='false'
-                            ref={this.setRef(provided.innerRef)}
-                            className={classNames('SidebarChannel', {
-                                collapsed: this.isCollapsed(this.props),
-                                unread: this.isUnread(),
-                                active: isCurrentChannel,
-                                dragging: snapshot.isDragging,
-                                fadeDMs: snapshot.isDropAnimating && snapshot.draggingOver?.includes('direct_messages'),
-                                noFloat: isDMCategory && !snapshot.isDragging,
-                            })}
-                            onTransitionEnd={this.removeAnimation}
-                            {...provided.draggableProps}
-                            {...provided.dragHandleProps}
-                            role='listitem'
-                            tabIndex={-1}
-                        >
-                            <ChannelComponent
-                                isCollapsed={this.isCollapsed(this.props)}
-                                channel={channel}
-                                currentTeamName={currentTeamName}
-                            />
-                        </li>
-                    );
-                }}
-            </Draggable>
+        const component = (
```

Comment:
```suggestion
        const channel = (
```
?
(feels more natural; not important, just a name)

Reply:
It would feel a bit nicer, but that conflicts with `const channel = this.props.channel` above